src/_scss/pages/search/_toggleTempSearchPage.scss

```diff
@@ -0,0 +1,47 @@
+// this file can be deleted once the new advance search has been implemented
+// and the toggle between new and old search ui is no longer needed
+
+.temp-search-page {
+  display: grid;
+}
```

Comment:
technically this curly bracket should be at the bottom of this page, since everything in this file is under this class

Reply:
This has been updated.